### PR TITLE
18NewEngland - ISR, SR fixes. First pass OR steps

### PR DESCRIPTION
**Pull request by roseundy**
* Fix issues with ISR (consecutive passing and snake order)
* Make market 2D to allow choice of top or bottom minor share value
* Tile upgrade rules for pre-printed track
* Tile 63 vs 611 rule
* 1/2 dividends
* minor train buying rules
* minor liquidation
* share redemption and issuing (not tested)

Common file change:
* UI:par - don't print parenthetical info on par values for companies that have 100% president shares

### CODE REVIEW

**Comment by scottredracecar** [comment]
From what I can tell, looks good!

**Review by scottredracecar — APPROVED** [review]

File: lib/engine/game/g_18_new_england/game.rb [hunk 1]
```diff
@@ -274,44 +334,55 @@ def buy_first_train(corporation)
 
         def minor_yellow_prices
           @minor_yellow_prices ||=
-            YELLOW_PRICES.map { |p| lookup_price(p) }
+            (YELLOW_PRICES.map do |p|
```
Comment (tobymao) [inline lib/engine/game/g_18_new_england/game.rb:337]: flat_map?

File: lib/engine/game/g_18_new_england/game.rb [hunk 2]
```diff
@@ -274,44 +334,55 @@ def buy_first_train(corporation)
...
+               [lookup_minor_price(p, TOP_MINOR_ROW), lookup_minor_price(p, BOTTOM_MINOR_ROW)]
+             end).flatten
         end
 
         def minor_green_prices
           @minor_green_prices ||=
-            GREEN_PRICES.map { |p| lookup_price(p) }
+            (GREEN_PRICES.map do |p|
```
Comment (tobymao) [inline lib/engine/game/g_18_new_england/game.rb:344]: flat_map

File: public/logos/18_new_england/CLOSED.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tobymao) [inline public/logos/18_new_england/CLOSED.svg:1]: compress this?

Comment (roseundy) [reply public/logos/18_new_england/CLOSED.svg]: OK

File: lib/engine/game/g_18_new_england/game.rb (hunk 2 above)
Comment (roseundy) [reply lib/engine/game/g_18_new_england/game.rb]: OK

File: lib/engine/game/g_18_new_england/game.rb (hunk 1 above)
Comment (roseundy) [reply lib/engine/game/g_18_new_england/game.rb]: OK

**Pull request merged**